artix7/primitives/common_slice/common_slice.pb_type.xml

```diff
@@ -38,26 +38,28 @@
   <input  name="CIN"  num_pins="1"/>
   <output name="COUT" num_pins="1"/>
 
-  <output name="DMUX" num_pins="1"/>
+  <output name="DOUTMUX" num_pins="1"/>
```

Comment:
Do not rename these, they were named correctly before.

Reply:
pacific time? I, most probably, missed the slot, but we can sync tomorrow. I live in CET time zone.

Reply:
I have restored the original output name and changed only metadata names for *MUXes. Please review this one